File: WebApp/src/app/person.ts
export interface Person {
  id: string;
  firstName: string;
  lastName: string;
  fullName: string;
  initials: string;
  hasPicture: boolean;
  pictureUrl: string | null;
  email?: string;
  phone?: string;
  address?: string;
  dateOfBirth?: string;
  company?: string;
  jobTitle?: string;
  notes?: string;
}

File: ServiceModel/Person.cs
namespace ServiceModel;

public class Person
{
    public required string Id { get; set; }
    public required string FirstName { get; set; }
    public required string LastName { get; set; }
    public required string FullName { get; set; }
    public required string Initials { get; set; }
    public bool HasPicture { get; set; }
    public string? PictureUrl { get; set; }
}
